src/Schemes/AffineSchemes/Objects/Constructors.jl

```diff
@@ -0,0 +1,670 @@
+export affine_space
+export closure
+export hypersurface_complement
+export standard_spec
+export subscheme
+
+
+########################################################
+# (1) Generic constructors
+########################################################
+
+@doc Markdown.doc"""
+    Spec(R::MPolyRing, I::MPolyIdeal)
+
+Constructs the affine scheme of the ideal ``I`` in the ring ``R``.
+This is the spectrum of the quotient ring ``R/I``.
+
+# Examples
+```jldoctest
+julia> R, (x, y) = PolynomialRing(QQ, ["x", "y"]);
+
+julia> I = ideal(R, [x]);
+
+julia> Spec(R, I)
+Spec of Quotient of Multivariate Polynomial Ring in x, y over Rational Field by ideal(x)
+```
+"""
+Spec(R::MPolyRing, I::MPolyIdeal) = Spec(quo(R, I)[1])
+
+
+@doc Markdown.doc"""
+    Spec(R::MPolyRing, U::AbsMPolyMultSet)
+
+Given a polynomial ring ``R`, we can localize that polynomial
+ring at a multiplicatively closed subset ``U`` of ``R`. The spectrum
```

Comment:
sideremark: Actually in literature it is customary to call the multiplicatively closed set S, not U.

Reply:
Wolfram changed this to U so that S could be a ring. We should stick to that.

Reply:
ok. Fine with me -- as long as it is consistent in the documentation

Reply:
To me the name makes sense as are. `U` indicates a set with units. Often `R`, `S` are used for rings, so this has the added advantage of leaving `S` free.

Reply:
It also makes sense to me, I just wanted to point out that in Textbooks one often finds S^-1 R for the localization.